composer.json

```diff
@@ -27,6 +27,9 @@
   "require": {
     "oat-sa/oatbox-extension-installer": "~1.1||dev-master"
   },
+  "require-dev": {
+    "php-mock/phpmock": "^2.0"
```

Comment:
I don't think is a good idea to add another library that will increase the maintenance costs,
I think the full purpose of this lib is to provide an easier way of mocking but  the PHPUnit gives you createMock

Reply:
In fact, it's simplifying the mock of global variable (with the global namespace fallback), it's not PHPUnit's createMock.

Reply:
+1, phpunit does not allow this, ok to use php-mock imo